### ISSUE TITLE
[Bug]: Can not control the Message window font size

### ISSUE THREAD

**Issue by objectx** [mallowigi/material-theme-issues#310]
### Describe the bug

After enabling the MUI.  Unable to change the message window font size (seems to use the default font size)

<img width="1394" alt="12" src="https://user-images.githubusercontent.com/1112949/221197632-2a447210-d240-440c-8910-d3a728ad99d1.png">
<img width="1370" alt="36" src="https://user-images.githubusercontent.com/1112949/221197921-92e3248e-94ce-4eda-8f7b-aa10d44ef43e.png">

### To Reproduce

 1. Enable MUI
 2. Change the console font size.
 

### Expected behavior

Message window font size reflects the setting.

### Material Theme Version

v8.5.0

### IDE

Other (AppCode, CLion, GoLang...)

### System Info

```shell
CLion 2023.1 EAP
Build #CL-231.7665.29, built on February 23, 2023
Licensed to CLion EAP user: Masashi Fujita
Expiration date: March 25, 2023
Runtime version: 17.0.6+10-b829.1 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.7
GC: G1 Young Generation, G1 Old Generation
Memory: 10000M
Cores: 20
Metal Rendering is ON
Registry:
    editor.focus.mode.color.light=6b727d
    ide.editor.tab.selection.animation=true
    debugger.new.tool.window.layout=true
    run.processes.with.pty=TRUE
    debugger.valueTooltipAutoShowOnSelection=true
    ide.new.editor.tabs.vertical.borders=true
    ide.tooltip.initialDelay=312
    ide.experimental.ui=true
    debugger.attach.dialog.enabled=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=6b727d

Non-Bundled Plugins:
    com.paperetto.dash (3.3)
    pw.unbear.ijplug (1.0.0)
    org.editorconfig.editorconfigjetbrains (231.7665.29)
    com.github.leomillon.uuidgenerator (4.5.1)
    GLSL (1.24)
    org.antlr.intellij.plugin (1.19.3-3)
    Batch Scripts Support (1.0.13)
    org.rust.lang (0.4.188.5205-231)
    com.nasller.CodeGlancePro (1.6.0-2023.1)
    org.nik.presentation-assistant (1.0.10)
    org.jetbrains.plugins.go-template (231.7665.25)
    com.intellij.ideolog (203.0.30.0)
    org.jetbrains.plugins.rest (231.7665.20)
    com.intellij.plugin.adernov.powershell (2.0.10)
    com.intellij.grazie.pro (0.3.207)
    com.chrisrm.idea.MaterialThemeUI (8.5.0)
    izhangzhihao.rainbow.brackets (2023.2.5)
    com.koxudaxi.ruff (0.0.9)
    com.mallowigi (87.1.0)
    zielu.gittoolbox (500.0.4+213)
    mobi.hsz.idea.gitignore (4.4.4)
    com.draculatheme.idea (1.4.0)
```

**Comment by objectx** [comment]
cf. https://youtrack.jetbrains.com/issue/CPP-32401#focus=Comments-27-6902353.0-0

**Comment by mallowigi** [comment]
I saw the thread and tried to reproduce, but it worked well as far as I can tell.

Is it happening on all themes?

**Comment by objectx** [comment]
@mallowigi Yes. All themes, including pre-installed JetBrains Light and Dark.
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/1112949/221371311-a39dd700-748b-4b47-8593-67b0d87c29c1.png">
<img width="922" alt="image" src="https://user-images.githubusercontent.com/1112949/221371370-b5ac789c-6674-4609-9d00-83e6190f173f.png">

**Comment by mallowigi** [comment]
Still cannot reproduce, or I didn't understand the issue. Can you provide the steps using a recording?

**Comment by objectx** [comment]
Quite strange… After stopping setting sync and erasing setting directory then reinstall, Same symptom happend 😱.

And I can't restore the intended behavior (console font size seems to be always ignored)

This seems not to be a MUI's issue.

**Comment by objectx** [comment]
Sorry for the confusion.

I've reset the settings via "Restore Default Settings...". Then install only the [Atom Material Icons](https://plugins.jetbrains.com/plugin/10044-atom-material-icons) plugin; the symptom happens! and even after uninstalling Atom Material Icons, the symptom never disappears.

**Comment by objectx** [comment]
The issue is resolved on 2023.1 RC.  (Related? > https://youtrack.jetbrains.com/issue/IDEA-314623 )
